web/templates/navbar.html

```diff
@@ -53,6 +53,16 @@
             </ul>
             {% if request.user.is_authenticated %}
                 <ul class="navbar-nav">
+                    <li class="nav-item {% if request.resolver_match.url_name == "notifications" %}active{% endif %}">
```

Comment:
should we disable/remove the notifaction button when NOTIFICATIONS_DISABLED is true?

Reply:
I think it would make sense yes. I will check how to change that

Reply:
Corrected

Reply:
I am not sure if we should do the same for notifications_settings and admin_notifcations, what do you think ?

Reply:
In the settings it is commented that 
```
# by default, notifications by email are disabled
NOTIFICATIONS_DISABLED = True
```

So it seems that that isn't for the in_app notifications.

Reply:
True, but it would still make sense to remove the buttons if the notifications are not enabled. I will also put these menus under a `if` condition

Reply:
works, thanks.